py-polars/polars/lazyframe/frame.py

```diff
@@ -1370,6 +1371,8 @@ def sort(
         """
         # Fast path for sorting by a single existing column
         if isinstance(by, str) and not more_by:
```

Comment:
I think we should just not hit the fast path if the user passes a list: 

`if isinstance(by, str) and isintance(descending, bool) and not more_by:`